lib/surveyor/models/question_group_methods.rb

```diff
@@ -15,7 +15,6 @@ module QuestionGroupMethods
         # Associations
         has_many :questions
         has_one :dependency
-        attr_accessible *PermittedParams.new.question_group_attributes if defined? ActiveModel::MassAssignmentSecurity
```

Comment:
@careful-axe it is worth reviewing this PR a bit yourself though. After all this time I am forgetting whether changes like this were a "no longer needed" or a "look at this later, but it breaks things now" 🥴